site/src/analytics-average.ts

```diff
@@ -36,11 +36,12 @@ export class AnalyticsAverage extends LitElement {
   static styles = css`
     .grid {
       display: grid;
-      grid-template-columns: repeat(2, 1fr);
-      grid-template-rows: repeat(2, 1fr);
+      grid-template-columns: repeat(4, 1fr);
```

Comment:
Add a minmax here, so it does nice responsive.

Reply:
mmmm that might not work well here, then you will get a row of 3 and 1 at some breakpoints.

Then just add an extra breakpoint to get to 2 columns?